Doc/library/typing.rst

```diff
@@ -421,8 +423,9 @@ Note that ``type[C]`` is covariant::
        return user_class()
 
    make_new_user(User)      # OK
-   make_new_user(ProUser)   # Also OK: ``type[ProUser]`` is a subtype of ``type[User]``
-   make_new_user(TeamUser)  # Still fine
+   make_new_user(BasicUser) # Also OK: ``type[BasicUser]`` is a subtype of ``type[User]``
```

Comment:
This now demonstrates the exact same thing three times. I think we only need one. We can keep one of the three subclasses in this test, and move the remaining two class definitions to the next test.